packages/plugin-conversation/src/conversation.js

```diff
@@ -453,6 +453,24 @@ const Conversation = SparkPlugin.extend({
     return ShareActivity.create(conversation, null, this.spark);
   },
 
+  assign(conversation, avatar) {
+    return this._inferConversationUrl(conversation)
+      .then(() => {
+        const activity = ShareActivity.create(conversation, null, this.spark);
+        activity.add(avatar);
+
+        return this.prepare(activity, {
+          target: this.prepareConversation(conversation)
+        });
+      })
+      .then((a) => {
```

Comment:
do we need to check that the file is <1MB? or is server supposed to do that? and makes sure that files.items.length is 1